services/vehicle-assembly-process-defect-detection-model-service/app/schemas/__init__.py

```
@@ -0,0 +1,219 @@
+    ServiceEndpoints,
+    ApiDocs
+)
+
+# === 헬스체크 스키마 ===
+from .health import (
+    # 헬스체크 데이터 모델들
+    HealthData,
+    ModelInfo,
+    ReadinessData,
+    StartupData,
+    StartupCheck,
```

Ruff (0.12.2)

`.health.StartupCheck` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)